src/AccessibilityInsights.DesktopUI/Highlighters/Win32SnapshotButton.cs

```diff
@@ -93,7 +93,7 @@ private static PrivateFontCollection LoadFontResource()
             PrivateFontCollection fonts = new PrivateFontCollection();
             Assembly assembly = Assembly.GetExecutingAssembly();
 
-            using (Stream fontStream = Application.GetResourceStream(new Uri(@"pack://application:,,,/AccessibilityInsights.DesktopUI;component/Resources/FabMDL2.ttf")).Stream)
+            using (Stream fontStream = Application.GetResourceStream(new Uri(@"pack://application:,,,/AccessibilityInsights.CommonUxComponents;component/Resources/FabMDL2.ttf")).Stream)
```

Comment:
>CommonUxComponents [](start = 124, length = 18)

It feels odd to me to "reach across" and extract resources from another assembly. Is there a way that we could explicitly expose the resource via code?

Reply:
Added a SharedResources static class to CommonUxComponents. Check it out!